host/4/bullseye/amd64/python/python310/python310-buildenv.Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM python:3.10-slim-bullseye
+FROM mcr.microsoft.com/mirror/docker/library/python:3.10-slim
```

Comment:
Bullseye Suffix image does not exist. 
This is not ideal but is not our responsibility. Just calling out.